Client/TeamTalkClassic/TeamTalkDlg.cpp

```diff
@@ -1,7182 +1,7177 @@
-/*
```

Comment:
Hm, there seems to be some line ending problem here. Everything is remove and added again.